tests/e2e/performance/mcg/test_list_objects.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.bucket_utils import copy_objects, craft_s3_command
+from ocs_ci.ocs.perftests import PASTest
+from ocs_ci.framework.testlib import performance, bugzilla, skipif_ocs_version
+from ocs_ci.ocs.resources.mcg import MCG
+
+
+log = logging.getLogger(__name__)
+
+
+@performance
+@bugzilla("2052079")
+@skipif_ocs_version("<4.8")
+@pytest.mark.polarion_id("OCS-3926")
+class TestListOfObjects(PASTest):
+    """
+    Test to verify the list whole objects without any failures
+
+    """
+
+    def test_list_large_number_of_objects(
+        self,
+        fedora_pod_session,
```

Comment:
Instead of creating separate fedora pod session use `awscli_pod_session` and then you can run the below command on pod as part of the test before line 45
```
command = f"mkdir {data_dir} && cd {data_dir} && \
    wget "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.4.tar.xz" && \
    tar -xvf linux-4.4.tar.xz && \
    for i in $(seq 1 10); do cp -r linux-4.4/ linux-$i; done"
```
This will make your code much shorter and more efficient

Reply:
OR we can achieve the same through a setup fixture where teardown takes care of the cleanup of files at the end of test.
wdyt? @MeridianExplorer